lib/view_model/active_record/update_context.rb

```diff
@@ -165,6 +165,10 @@ def run!(deserialize_context:)
 
       @release_pool.release_all!
 
+      if updated_viewmodels.present?
```

Comment:
I believe that's the case. That said, updated_viewmodels should always include all of the roots, whether changes were made or not, so the `present?` check is really only guarding against an empty operation.